install-dependencies.sh

```diff
@@ -121,6 +116,17 @@ fedora_python3_packages=(
     python3-six
 )
 
+pip_packages=(
+    'geomet<0.3,>=0.1'
+    boto3
+    pytest
+    pytest-asyncio
+    redis
+    aiohttp
+    traceback-with-variables
+    scylla-api-client
```

Comment:
Maybe we should specify the versions or version ranges to make the experience more stable?

Reply:
pinning them in some manner would be wise. other wise each time someone would install or build dbuild it might get a surprise by a package not being backward compatible, or just plain bugs.

Reply:
Let's not do it. That will make things works for people like myself with Ubuntu.

Reply:
@kostja why?

Reply:
Because there may be conflicts with other projects which require different versions. fixing versions = virtualenv

Reply:
Ah, because you cannot have multiple versions of the same package installed globally.
But you can also use virtualenv for those other projects :P

Reply:
it's virtualenvs all the way down 🕳️ 

I even use this:
```
export PIP_REQUIRE_VIRTUALENV=true
```

to make sure never to pip install into the system python

Reply:
We already are pinning versions by requiring a certain build environment in dbuild. So fix it in pip it's twice. Those like me who don't use dbuild, bring their own build, will have two hoops to jump through. Not fixing the dependencies at least gives it a chance that the code won't break in a wider range of environment.